build/drivers/dfu_trigger_and_cdc_acm/nrfconnect_dfu_trigger.inf

```diff
@@ -0,0 +1,176 @@
+Provider    = %VendorName%
+CatalogFile = nrfconnect_dfu_trigger.cat
+DriverVer   = 11/17/2017,1.0.0.0
+
+; ============== Class section ==================
+[ClassInstall32]
+Addreg = Nordic_Class_AddReg
+
+[Nordic_Class_AddReg]
+HKR,,,0,"Nordic Semiconductor DFU"
+HKR,,Icon,,-20
+
+; ============== Device section =================
+[Manufacturer]
+%VendorName% = Nordic_DeviceGroup,NTX86,NTAMD64
+
+; For Win2K
+[Nordic_DeviceGroup]
+%DeviceName% = Nordic_Device, USB\%VendorID%, USB\%CompatID%
+
+; For XP and later
+[Nordic_DeviceGroup.NTX86]
+%DeviceName% = Nordic_Device, USB\%VendorID%&PID_C00A&MI_00,USB\COMPOSITE
+
+[Nordic_DeviceGroup.NTAMD64]
+%DeviceName% = Nordic_Device, USB\%VendorID%&PID_C00A&MI_00,USB\COMPOSITE
+
+[Nordic_Device.NT.HW]
+AddReg=Nordic_Device_AddReg
+
+[Nordic_Device.NTAMD64.HW]
+AddReg=Nordic_Device_AddReg
+
+[Nordic_Device_AddReg]
+HKR,,DeviceInterfaceGUIDs,0x10000,%DeviceGUID%
+
+; ======= Power policy settings (default) ========
+
+; Indicates whether the device is capable of being powered down when idle
+; (Selective Suspend).
+HKR,,DeviceIdleEnabled,0x00010001,1
+
+; Sets the default value of the AUTO_SUSPEND power policy setting. This
+; registry key is used to enable or disable selective suspend when a
+; handle is not open to the device.
+HKR,,DefaultIdleState,0x00010001,0
+
+; Sets the default state of the SUSPEND_DELAY power policy setting. The
+; value indicates the amount of time in milliseconds to wait before
+; determining that a device is idle.
+HKR,,DefaultIdleTimeout,0x00010001,5000
+
+; When set to a nonzero value, it suspends the device even if it does not
+; support RemoteWake.
+HKR,,DeviceIdleIgnoreWakeEnable,0x00010001,0
+
+; Indicates whether a check box should be enabled in the device Properties
+; page that allows a user to override the idle defaults.
+HKR,,UserSetDeviceIdleEnabled,0x00010001,0
+
+; Indicates whether the device should be allowed to wake the system from a
+; low power state.
+HKR,,SystemWakeEnabled,0x00010001,0
+
+; ========== Driver/Service setup ==============
+[Nordic_Device.NT.Services]
+Addservice=libusbK,2,LUsbK_AddService
+
+[Nordic_Device.NTAMD64.Services]
+Addservice=libusbK,2,LUsbK_AddService
+
+[LUsbK_AddService]
+DisplayName="Nordic Semiconductor DFU USB Driver 17/11/2017 - 1.0.0"
+ServiceType=1
+StartType=3
+ErrorControl=1
+ServiceBinary=%12%\libusbK.sys
+AddReg=LUsbK_Driver_AddReg
+
+; This optional entry identifies the load order group of which this driver is a member.
+; LoadOrderGroup=Base
+
+[LUsbK_Driver_AddReg]
+HKR,"Parameters","DebugLevel",0x10001,4
+
+; =========== Sources/Destinations ==============
+[SourceDisksNames]
+1 = %SourceName%
+
+[SourceDisksFiles.x86]
+libusbK.sys	= 1,x86
+libusbK_x86.dll	= 1,x86
+libusb0_x86.dll	= 1,x86
+WdfCoInstaller01009.dll = 1,x86
+
+[SourceDisksFiles.amd64]
+WdfCoInstaller01009.dll	= 1,amd64
+libusbK.sys			= 1,amd64
+libusbK.dll			= 1,amd64
+libusb0.dll			= 1,amd64
+libusb0_x86.dll		= 1,x86
```

Comment:
Is it correct to use x86 here in the amd64 section?

Reply:
I *guess* that this refers to the file to be used when running stuff in 32-bit mode in a 64-bit system, so it makes sense to have both 64-bit and 32-bit entries in the 64-bit file list

Reply:
Ok, sounds sensible.